__tests__/app.test.js

```diff
@@ -42,7 +42,7 @@ describe('GET /api/topics', () => {
 })
 
 describe('GET /api/articles/:article_id', () => {
-  test('status:200, responds with single matching article object', () => {
+  test('status:200, matching article object with comment_count', () => {
```

Comment:
shouldn't have changed old test here, should add new test that asserts new property and the correct value